### ISSUE TITLE
Add aria tags to modal dialogs

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#496]
Make sure all modal dialogs are using the proper Aria tags.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role

Search for `modal` across the views to find the appropriate files. Also make sure that they follow the same pattern as the one found in the `views/layout/_signin_signout.html.erb`. 
This uses `data-toggle="modal"` attribute to signify that the link will open a modal and the `href` tag to indicate which div to display as the modal. That div should have the `class="modal"` attribute and a `style="display: none;"`

For example:
```html
<a href="#header-login-form" data-toggle="modal" dialog-width="450px"><%= _('Sign in')%></a> /
<a href="#header-register-form" data-toggle="modal" dialog-width="450px"><%= _('Create account')%></a>

<div id="header-login-form" class="modal" style="display:none">
  <%= render :partial => "shared/login_form" %>
</div>
<div id="header-register-form" class="modal" style="display:none">
  <%= render :partial => "shared/register_form", locals: {extended: false} %>
</div>
```

**Comment by xsrust** [comment]
above commit should address all modals within the site.

**Comment by stephaniesimms** [comment]
Unless @JEK-III is able to test this? i'll close the issue and await next round of accessibility testing.
